content/docs/faq-functions.md

```diff
@@ -201,18 +201,17 @@ class LoadMoreButton extends React.Component {
 #### Debounce
 
 ```jsx
-import { debounce } from 'lodash'
+import debounce from 'lodash.debounce'
 
 class Searchbox extends React.Component {
   handleChange = event => {
     event.persist()
-    this._handleChangeDebounced(event)
+    this._handleChangeDebounced(event.target.value)
```

Comment:
Now that you're doing this, you no longer need to call `event.persist()` 😄

Reply:
Ah right... I think the event.persist() example is important. It's easy to forget and causes quite the headache when debouncing forms. What do you think about reverting this change and adding a note about capturing `value` as an alternative to `.persist` above?